Makefile

```diff
@@ -55,6 +55,7 @@ codegen:
 	${OPENAPI_GENERATOR} generate -g go \
 		--package-name ${PACKAGE_MAJOR} \
 		--http-user-agent "${GIT_REPO}/${PACKAGE_VERSION}" \
+		--api-name-suffix Api \
```

Comment:
The new version of openapi-generator includes a breaking change; previously API Go struct names ended with `Api`, but starting with v7.0.0 they end with `API` by default.  This flag was added to enable keeping the old behavior.  I've set the flag here, but given that this repo is experimental it may be preferable to accept the breaking change.